finance/proposal-calls/internal_template.md

```diff
@@ -0,0 +1,56 @@
+
+## Proposal goals
+Proposals shall improve the Astropy Project. This includes code for the core repository, for coordinated or affiliated packages, infrastructure (e.g. testing, CI), tutorials, workshops, notebooks, community engagement, inclusion, diversity and empowerment, and anything else broadly Project related. 
+
+Proposals shall foster the Astropy Project goals of community, shared work, and cooperation and shall be proposed and executed in this manner.
+
+Total funding for this call is coming from (**link to grant here**) and XXX $ are available in total over XX months. 
+
+Proposals need to clearly identify what the funding is used for (e.g. "pay YYY $ to add Klingon Standard Time to astropy.time" or "organize astropy workshop to engage amateur astronomer community in Antarctica")
+
+## Proposal details
+**Fill in here when using this template to issue an internal Astropy call**
+
+- Source of money (e.g. Moore grant)
+- Specific area of proposal (e.g. Astropy may have a grant that can be used only for a specific purpose, e.g. only for community outreach)
+- Restrictions: Depending on who provided the money to astropy, it’s use might be restricted, e.g. only to contractors in certain countries
+
+## Proposal process
+Proposals should be posted as pull requests to github.com/astropy/astropy-project/ with the initial title text "Phase II Proposal:".
+The goal is to make this process as easy as possible on everyone, so we impose a limit of 100 lines assuming 80 char/line. We provide a template below. 
+We envision a two stage proposal process to enable the community to comment and improve ideas, teams with similar ideas can merge to avoid duplicated work, etc...
+
+- Deadline 1: Draft Proposal Deadline (after this date, no new proposals)
+- Two week discussion period during which proposals can be modified
+- Deadline 2: Finalized Proposals are due
+- Finance Committee reviews for compatibility with legal conditions
+- Deadline 2 + 7 business days: voting begins.
+
+*Rationale: We aim to achieve community consensus before the vote. Proposers may combine ideas and teams or withdraw proposals during the discussion period.*
+
+At the second  deadline, the text for the proposals is frozen but comments are still allowed. Within 7 days of this deadline, the Finance Committee will review all proposals to ensure they are compatible with the terms of the currently available funding sources and should be included in the proposals to be voted on.
+
+After that, voting members of the Astropy Project vote on the proposals. The vote is held online with a secret ballot. Every voting member of the Astropy Project (as defined in the governance documents) gets X votes they can assign to one or more proposals. After the vote, proposals are ranked according to the number of votes.
```

Comment:
That was the idea, so you could do 3 votes to proposal A (because it's really important to you) and 1 vote to C and D each (because those are good, certainly better than B, but not as good as A). But, as you say below, we many not need to sweat the details at this point.